FileParser.java

```diff
@@ -8,6 +8,9 @@
 
 public class FileParser {
     public static Optional<List<String>> getWordList(String filePath) throws IOException {
+        if (filePath == null) {
+            return Optional.empty();
+        }
```

Comment:
Can the caller recovered from this? Should we raise a runtime explicit/implicit error?